prboom2/src/MUSIC/portmidiplayer.c

```diff
@@ -178,6 +178,32 @@ static int pm_init (int samplerate)
   return 1;
 }
 
+#ifndef MIDI_EVENT_CHANNEL_MODE
+
+// channel mode events in MIDI, low nibble is channel value as usual
+#define MIDI_EVENT_CHANNEL_MODE 0xB0
+
+#endif
+
+static void pm_killnotes (unsigned long when = 0)
```

Comment:
C language doesn't support default values, it's a C++ thing.

Reply:
Yeah, I noticed that with the CI failure. I was just pushing commits to fix that :p